fgpyo/sam/__init__.py

```diff
@@ -1266,8 +1190,220 @@ class SamOrder(enum.Enum):
 
 
 @dataclass(frozen=True)
-class SecondaryAlignment:
-    """A secondary alignment as encoded in one part of the `XA` or `XB` tag value on a SAM record.
+class AuxAlignment(ABC, DataclassInstance):
```

Comment:
Yeah, this is a better name than `AlternateAlignment` since it's shorter and wont be confused as an alignment to an _alternate_ contig.